fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/executor/Rewriter.java

```diff
@@ -405,6 +406,8 @@ private static List<RewriteJob> getWholeTreeRewriteJobs(List<RewriteJob> jobs) {
                 topic("rewrite cte sub-tree",
                         custom(RuleType.REWRITE_CTE_CHILDREN, () -> new RewriteCteChildren(jobs))
                 ),
+                topic("or expansion",
```

Comment:
what is the reason to put it here?

Reply:
Because `or expansion` could generate new CTE, it may break the assumption that the CTE anchor always is at the top of the plam